app/components/post_list/thread_overview/__snapshots__/thread_overview.test.tsx.snap

```diff
@@ -94,6 +102,14 @@ exports[`ThreadOverview should match snapshot when post is saved and has replies
       onGestureHandlerStateChange={[Function]}
       onHandlerStateChange={[Function]}
       rippleColor={0}
+      style={
+        [
+          undefined,
+          {
+            "cursor": undefined,
+          },
+        ]
+      }
```

Comment:
A bit surprised of these snapshots changing. Do you know where is this change coming from?

Reply:
this can be caused by the upgrade of react-native-gesture-handler library or the update to react-native or react, this is defo not done by us.